meta-resin-common/recipes-support/resin-vars/resin-vars/config-json.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Config.json watcher
+Requires=mnt-boot.mount
+After=mnt-boot.mount
+
+[Service]
+Type=oneshot
+ExecStart=/bin/echo 'config.json changed'
+
+[Install]
+WantedBy=mnt-boot.mount
```

Comment:
As well, this will be activated by mnt-boot.mount too even if the config json was not changed. Still a question mark about loop dependency.